docs/can-guides/commitment/recipes/file-navigator/file-navigator-simple.md

```diff
@@ -350,14 +370,36 @@ const Entity = can.DefineMap.extend("Entity", {
 
 const rootEntity = new Entity(rootEntityData);
```

Comment:
I don’t think the code works after this section because `rootEntity` isn’t used where.

Reply:
Done!